docs/guides/ros2-gem.md

```diff
@@ -88,6 +90,15 @@ You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot
 
 It is possible to implement your own control mechanisms with this Component.
 
+#### Spawner
+
+`ROS2SpawnerComponent` handles spawning entities during simulation.
```

Comment:
add example of service call to spawn a robot please

Reply:
I changed the docs in the [robot spawner PR](https://github.com/RobotecAI/o3de-ros2-gem/pull/172) since it is related to spawner itself and not the default spawn point.